Cabal/doc/developing-packages.rst

```diff
@@ -2550,8 +2551,14 @@ system-dependent values for these fields.
     command-line arguments with the :pkg-field:`cc-options` and the
     :pkg-field:`cxx-options` fields.
 
+.. pkg-field:: cmm-options: token list
```

Comment:
The thing that worries me here is that this should rather be called `ghc-cmm-options`, because we don't really have an actual C-- compiler -- we're just invoking GHC; and `cmm-options` would imply that the flags you can pass to the `C--` compiler are somewhat universal to C-- compilation.

Reply:
I would keep `cmm-options` here. Mainly for consistency and also because the fact that we use GHC as a `C--` compiler is incidental and might change in the future if we split GHC into smaller components.

Reply:
Agreed that `ghc-cmm-options` would be a better name.

Reply:
This ship has unfortunately sailed.